### PR TITLE
Adding optional buffer FileSet class and related predicates

### PR DESCRIPTION
**Pull request by escowles**
Objects can either link directly to Files, or link to FileSets that link to Files:
- Current PCDM:
  
  ```
  <object1> a pcdm:Object ;
  pcdm:hasFile <file1> .
  
  <file1> a pcdm:File .
  ```
- Optional FileSet as a buffer between Object and File:
  
  ```
  <object1> a pcdm:Object ;
  pcdm:hasFileSet <fileset1> .
  
  <fileset1> a pcdm:FileSet ;
  pcdm:hasFile <file1> .
  
  <file1> a pcdm:File .
  ```

See also #69

### CODE REVIEW

File: models.rdf [hunk 1]
```diff
@@ -16,8 +16,8 @@
       <dcterms:publisher rdf:resource="http://www.duraspace.org/"/>
       <rdfs:seeAlso rdf:resource="https://github.com/duraspace/pcdm/wiki"/>
       <rdfs:comment xml:lang="en">Ontology for the Portland Common Data Model, intended to underlie a wide array of repository and DAMS applications.</rdfs:comment>
-      <owl:versionInfo>2016/04/18</owl:versionInfo>
-      <owl:priorVersion rdf:resource="http://pcdm.org/2015/09/28/models"/>
+      <owl:versionInfo>2016/09/01</owl:versionInfo>
```
Comment (awoods) [inline models.rdf:19]: And this: https://github.com/duraspace/pcdm/blob/27a133e2f5cdbf23cf0b2c125b2eb7f086c418bb/models.rdf#L15

**Comment by awoods** [comment]
:+1: pending update to: https://github.com/duraspace/pcdm/blob/27a133e2f5cdbf23cf0b2c125b2eb7f086c418bb/models.rdf#L15

**Comment by tpendragon** [comment]
I like this, but I do wonder (slightly) about the semantics of the fileset being the thing with the file, since there's no transitivity here.

**Comment by escowles** [comment]
@tpendragon Since FileSet is a subclass of Object, it's just another layer of Objects with a File.  So if your app doesn't care about FileSets, then it can just treat it as an Object.

**Comment by tpendragon** [comment]
Oh! I'm not sure FileSet in this proposal should be an object. Doesn't this open up FileSets having FileSets?

**Comment by cmharlow** [comment]
Yes, and doesn't help avoiding conflation of what metadata goes on Object (work, RWO) and Fileset. But in this change having Fileset be a subclass of Object means you don't need to create a new 'manages' property.

**Comment by azaroth42** [comment]
Collections could hasMember FileSets if they're subClassOf Object. That seems ... wrong.
From meh to :-1: given this.

**Comment by escowles** [comment]
I agree with the concerns raised here about FileSets being subclasses of Object causing problems.  Closing this issue in favor of #69, where we can continue to discuss what the relationships should look like and what metadata goes where.

**Pull request closed without merging**